packages/forms/src/UIForm/fields/MultiSelectTag/MultiSelectTag.component.js

```diff
@@ -139,6 +140,9 @@ export default class MultiSelectTag extends React.Component {
 		this.props.onChange(event, payload);
 		this.props.onFinish(event, payload);
 		this.updateSuggestions('');
+		setTimeout(() => {
+			this.resetSuggestions();
```

Comment:
Can you explain why you have this timeout ?

Reply:
Without, component seem to not re-render and suggestions remain visible.

Reply:
I have a feeling that doing both `this.updateSuggestions` and  `this.resetSuggestions` might cause bugs because they both setState on the same keys (focusedItemIndex and suggestions), plus one `setState` uses a function with oldState while the other does not. 

Any chance it's possible to simply skip `this.updateSuggestions` here? It feels like it's trying to reset them anyway right? That might allow you to remove the `setTimeout`.

Reply:
Actually I see that `onChange` *also* call `updateSuggestions` making the setState run 3 times. 😱

Reply:
Wanted to make as less changes as possible, but I can take a look ;)